.vscode/tasks.json.old

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
just delete it ...

Reply:
Omg, thought I did...

Reply:
Sorry for wasting your time on the first review.

Reply:
no worries, only takes a few seconds, thx for making a PR :)